### PR TITLE
Add sorting array performance in ruby

### PR DESCRIPTION
**Pull request by SantonyChoi**
I learned when sorting an array in descending order, there can be performance differences depending on my implementation.

### CODE REVIEW

File: ruby/sorting_array_performance_in_ruby.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Here's my result from the script.
+```
+                           user     system      total        real
+sort                   0.390000   0.010000   0.400000 (  0.398573)
+sort reverse           0.400000   0.000000   0.400000 (  0.397368)
+sort_by -a[:bar]       0.190000   0.000000   0.190000 (  0.195783)
+sort_by a[:bar]*-1     0.190000   0.000000   0.190000 (  0.190759)
+sort_by.reverse!       0.190000   0.000000   0.190000 (  0.195162)
+```
```
Comment (jeongwooklee) [inline ruby/sorting_array_performance_in_ruby.md:32]:
````suggestion
```

Aug 19 2019, by @SantonyChoi
````
😅

File: ruby/sorting_array_performance_in_ruby.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+You can sort an array in a various way. Just keep in mind that there is a difference in performance.
```
Comment (jeongwooklee) [inline ruby/sorting_array_performance_in_ruby.md:1]: Please update README.md as well
(e.g: https://github.com/Tapjoy/til/pull/17/files#diff-04c6e90faac2675aa89e2176d2eec7d8R45)

Comment (SantonyChoi) [reply ruby/sorting_array_performance_in_ruby.md:1]: Updated README.md
Thanks for the review 👍

Comment (jeongwooklee) [reply ruby/sorting_array_performance_in_ruby.md:1]: 👍

**Review by jeongwooklee — APPROVED** [review]
LGTM 👍

**Pull request merged**